### ISSUE TITLE
Fix Navigation

### ISSUE THREAD

**Issue by abidishajia** [the-collab-lab/tcl-8-smart-shopping-list#40]
If the token doesn't exist, the navigation doesn't appear at the bottom. The user has to refresh the page after the token has been saved in order for the navigation to show.

**Comment by segdeha** [comment]
Yep, noticed this. Happens for me only the first time when I create a new list
